common/Resources/ti.internal/extensions/binding.js

```diff
@@ -32,8 +32,8 @@ function isHijackableModuleId(path) {
 	return firstChar !== '.' && firstChar !== '/';
 }
 
-// Hack require to point to this as a core module "binding"
-const originalRequire = global.require;
+// Hack require to point to this as core module "binding". (Note that iOS does not have a global require.)
+const originalRequire = global.require ? global.require : require;
```

Comment:
This came up in a liveview change @ewanharris had worked on. I think he changed to using something like Module.main.require as the ultimate fallback, which makes sense to me.

Reply:
I agree. I should be able to access it via `require.main.require`. I'll make the change.
https://github.com/appcelerator/titanium_mobile/blob/master/common/Resources/ti.internal/kernel/module.js#L520